change view class

baptiste-mrtn opened the pull request:
Added some methods for displaying in console

RMartinCampus commented on src/fr/lecampusnumerique/cda25/javaalgo/squaregame/View.java:
```diff
@@ -21,23 +21,37 @@ public void setLines(Cell[][] board) {
-                System.out.println("Joueur1 VS Joueur2");
-            }
-        }
+    public void displayNbPlayerChoice(){
```
On ne sait pas si l'utilisateur saisit le nombre de joueur artificiel ou humain. Cette ambiguitée pose problème pour initialisé la partie (InteractionPlayer.getNbNPC() retourne le nombre de NPC ça serait mieux que le message demande le nombre de NPC.

RMartinCampus commented on src/fr/lecampusnumerique/cda25/javaalgo/squaregame/View.java:
```diff
@@ -21,23 +21,37 @@ public void setLines(Cell[][] board) {
+    }
+
+    public void displayTied(){
+        System.out.println("Game Over ! vous êtes nuls ...");
```
C'est honteux

RMartinCampus commented on src/fr/lecampusnumerique/cda25/javaalgo/squaregame/View.java:
```diff
@@ -21,23 +21,37 @@ public void setLines(Cell[][] board) {
+    }
+
+    public void displayErrorNbPlayerChoice(){
+        System.err.println("Nombre invalide ! Veuillez recommencer !");
```
Message incomplet, il serait bien de lui expliciter les valeurs attendu

RMartinCampus reviewed (changes requested):
1-2 fix mineur à faire, beau travail bg

baptiste-mrtn committed:
éFixes for reviews

baptiste-mrtn committed:
échange name of method + secific errors

RMartinCampus committed:
Fix : inverse l'orde des proposition displayNbPlayerChoice
Comme la fonction qui recupère l'input represente le nombre de NPC il est plus logique que Com vs Com correspond à 2 et JvJ à 0

RMartinCampus committed:
Feat : Ajout de la methode displayTypefOfPlayer
> Ajoute public void displayTypefOfPlayer(Symbol pionSymbol) à la classe et transformation de la classe en enum en prévision de la suite
